src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -31,7 +33,10 @@ const TopicFragment = ({
   setIsFirstTimeOpen,
   setSearchText,
   setFollowedTopic,
-  setUnfollowedTopic
+  setUnfollowedTopic,
+  fetchData = () => {},
+  searchText,
+  withoutRecent = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props are added to the component. Make sure these props are passed correctly where this component is used.
```diff
-  setUnfollowedTopic
+  setUnfollowedTopic,
+  fetchData = () => {},
+  searchText,
+  withoutRecent = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->